packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1341,12 +1336,24 @@ class SelectableRegionState extends State<SelectableRegion>
 
   /// Shows the text selection toolbar.
   ///
-  /// If the parameter `location` is set, the toolbar will be shown at the
-  /// location. Otherwise, the toolbar location will be calculated based on the
-  /// handles' locations. The `location` is in the coordinates system of the
-  /// [Overlay].
+  /// The toolbar location will be calculated based on the
+  /// location of the selection endpoints.
```

Comment:
We discussed offline already, but seeing the code reinforces my thoughts that this is a good approach. I think we're fine to avoid the `location` parameter. I don't want to spend any time supporting the deprecated TextSelectionControls approach.